ladon.go

```diff
@@ -66,57 +80,78 @@ func (l *Ladon) DoPoliciesAllow(r *Request, policies []Policy) (err error) {
 
 	// Iterate through all policies
 	for _, p := range policies {
-		// Does the action match with one of the policies?
```

Comment:
Is there any particular reason for altering the matching order?

Reply:
There was absolutely not -  I was experimenting with framework and must have not notice Im in the PR branch